checkers-app/src/components/vote/VotingTagChart.tsx

```diff
@@ -0,0 +1,108 @@
+import { PieChart, Pie, ResponsiveContainer, Cell, Legend } from "recharts";
+import { AssessedInfo } from "../../types";
+import { Typography } from "@material-tailwind/react";
+
+interface VotingTagChartProps {
+    assessedInfo: AssessedInfo | null;
+  }
+export default function VotingTagChart(Props: VotingTagChartProps)  {
+
+     const assessedInfo = Props.assessedInfo;
+     console.log(assessedInfo)
+     const totalResponse = Props.assessedInfo?.responseCount
+     const Incorrect_Tag_Data =
+        assessedInfo && totalResponse
+            ? [
+                { name: 'Tagged as Incorrect', value: assessedInfo.tagCounts.incorrect },
+                {
+                name: 'Not Tagged as Incorrect',
+                value: totalResponse - assessedInfo.tagCounts.incorrect,
+                },
+            ]
+            : [];
+
+     const Generated_Tag_Data = 
+            assessedInfo && totalResponse 
+                ? [
+                    {name: 'Tagged as Generated', value: assessedInfo.tagCounts.generated},
+                    {
+                        name: 'Not Tagged as Generated',
+                        value: totalResponse - assessedInfo.tagCounts.generated
+                    }
+                ]
+                : [];
+    
+    
+    const COLORS = ['#FFBB28', '#FF8042'];
```

Comment:
Change these 2 hex color codes to a more readable const? example, 'orange' or 'reddishOrange' etc